src/Pages/Competition.jsx

```diff
@@ -1,26 +1,25 @@
 import { useParams, useNavigate } from "react-router-dom";
 import { useEffect, useState } from "react";
 
-function Competition(props) {
+function Competition({ dogs, competitions }) {
+  const navigate = useNavigate();
 
-    const navigate = useNavigate();
-    function handleClick (event, ticket) {
-      event.preventDefault()
-      navigate(`/tickets`)
-    }
+  function handleClick(event) {
+    event.preventDefault();
+    navigate(`/tickets`);
+  }
 
-    function handleClick2 (event, ticket) {
-      event.preventDefault()
-      navigate(`/register`)
-    }
-  
-  const dogs = props.dogs;
-  const competitions = props.competitions;
+  function handleClick2(event) {
+    event.preventDefault();
+    navigate(`/register`);
+  }
 
+  // const dogs = props.dogs;
+  // const competitions = props.competitions;
 
-  console.log("props inside competition: ", props);
-  console.log("dogs in competition: ", dogs);
-  console.log("competitions in Competition", competitions);
+  // console.log("props inside competition: ", props);
+  // console.log("dogs in competition: ", dogs);
+  // console.log("competitions in Competition", competitions);
```

Comment:
Might want to consider removing this commented code if it is no longer needed :)